### PR TITLE
Improve CLI

### PR DESCRIPTION
**Pull request by RouquinBlanc**
Most of the time services are local domain. And even on local domain, I've not seen many `_udp` services.

Your CLI command is awesome, and in some aspects works better than the default mac OS CLI 👍 

With this changes, it will be more user friendly to use:
```
# The previous way of using (still works)
python -m aiozeroconf -s _ssh._tcp.local.

# With direct command line
aiozeroconf -s _ssh._tcp.local.

# Then assuming local domain
aiozeroconf -s _ssh._tcp

# And even assuming tcp
aiozeroconf -s _ssh
```

### CODE REVIEW

File: aiozeroconf/__main__.py [hunk 1]
```diff
@@ -66,50 +66,67 @@ def on_service_state_change(zc, service_type, name, state_change):
...
 
 
+def guess(service):
+    """
+    Attempt guessing and completing service name.
+    Most services are on _tcp, and even more on local domain!
+    """
+    if '.' not in service:
+        return service + '._tcp.local.'
+    elif service.endswith('._tcp') or service.endswith('._udp'):
```
Comment (42B) [inline aiozeroconf/__main__.py]:
```python
elif service.endswith(('._tcp', '._udp')):  # ?
```

Comment (RouquinBlanc) [reply aiozeroconf/__main__.py]: indeed! 👍

**Comment by frawau** [comment]
The "Automatic completion" isn't really necessary since the "find"  option exists. But ok.

**Comment by frawau** [comment]
I forgot... Thanks ;)

**Pull request merged**